requirements/edx/base.in

```diff
@@ -62,7 +62,6 @@ django-user-tasks
 django-waffle==0.12.0
 django-webpack-loader               # Used to wire webpack bundles into the django asset pipeline
 djangorestframework-jwt
-django-xforwardedfor-middleware==2.0 # Middleware to use the X-Forwarded-For header as the request IP
```

Comment:
Why are we removing the library?

Reply:
It  only set `REMOTE_ADDR`  ( https://github.com/allo-/django-xforwardedfor-middleware/blob/master/x_forwarded_for/middleware.py ) and  we also need to update SERVER_NAME and SERVER_PORT from xforwarded headers.
SAML authentication failed when we last deploy the change as it build request with SERVER_PORT and that was not override and exposing 8000 in response.